src/SpectatorView.Native/SpectatorView.Compositor/Compositor/IFrameProvider.h

```diff
@@ -39,4 +50,12 @@ class IFrameProvider
     virtual int GetCaptureFrameIndex() { return 0; }
     virtual int GetPixelChange(int frame) { return 0; }
     virtual int GetNumQueuedOutputFrames() { return 0; }
+    virtual bool IsCameraCalibrationInformationAvailable() { return false; }
+    virtual void GetCameraCalibrationInformation(CameraIntrinsics* calibration) {}
+
+    virtual bool IsArUcoMarkerDetectorSupported() { return false; }
```

Comment:
>IsArUcoMarkerDetectorSupported [](start = 17, length = 30)

It seems like every rgb camera should theoretically be able to support aruco marker detection. Should we consider an abstract base class where this is impleemented?